AndroidRunner/Adb.py

```diff
@@ -139,3 +140,15 @@ def logcat(device_id, regex=None):
         params += ' -e %s' % regex
     adb.set_target_by_name(device_id)
     return adb.get_logcat(lcfilter=params)
+
+
+def cleanup(cmd):
+    from time import sleep
```

Comment:
Move all `import` statements at the beginning of the files. See [here](https://www.python.org/dev/peps/pep-0008/#imports).